librocksdb-sys/src/patches.rs

```diff
@@ -0,0 +1,86 @@
+/* automatically generated by rust-bindgen 0.54.1 */
```

Comment:
rust-bindgen version is not matched with 
https://github.com/nervosnetwork/rust-rocksdb/blob/txn/librocksdb-sys/src/bindings.rs#L1
I suggest upgrade both or just  keep identical

Reply:
The `bindgen 0.53.2` will generate same code as `bindgen 0.54.1`.

So, I just remove the version in this comment. In fact, this version was added by me, not generated.
https://github.com/nervosnetwork/rust-rocksdb/blob/c6b2887b2f708c4e1aec6699fb031157d54a21ff/librocksdb-sys/src/patches.rs#L1
The command which used to generate the code was written in the `Makefile`. 
https://github.com/nervosnetwork/rust-rocksdb/blob/c6b2887b2f708c4e1aec6699fb031157d54a21ff/librocksdb-sys/Makefile#L8-L16

The first two lines of `binding.rs` was not generated, too.
https://github.com/nervosnetwork/rust-rocksdb/blob/686cdff2e4946c1b3d396e07acdabcec782de2b2/librocksdb-sys/src/bindings.rs#L1-L5